service/controller/key/cilium.go

```diff
@@ -10,3 +10,13 @@ func ForceDisableCiliumKubeProxyReplacement(cluster apiv1beta1.Cluster) bool {
 
 	return found && v == "true"
 }
+
+func AWSEniModeEnabled(cluster apiv1beta1.Cluster) bool {
```

Comment:
Can we rename the function? Just `CiliumEniModeEnabled or something?

If I see it right we check later in the code the provider, so I would be more general here.

Reply:
ups sorry didnt see the comment, I can rename it will open a different PR